lib/platform/bitbucket-server/index.ts

```diff
@@ -623,8 +626,13 @@ export async function getPrList(_args?: any) {
   logger.debug(`getPrList()`);
   // istanbul ignore next
   if (!config.prList) {
+    const query = new URLSearchParams({
+      state: 'ALL',
+      'role.1': 'AUTHOR',
+      'username.1': config.username,
```

Comment:
are `role.1` and `username.1` legit query params?

Reply:
Yes, the web ui is using the same query. [docu](https://docs.atlassian.com/bitbucket-server/rest/5.0.0/bitbucket-rest.html#idm45659054385280)

You can add multiple, up to ten filters.

Reply:
good